fs/fs_zenfs.h

```diff
@@ -452,6 +456,12 @@ class ZenFS : public FileSystemWrapper {
   IOStatus MigrateFileExtents(
       const std::string& fname,
       const std::vector<ZoneExtentSnapshot*>& migrate_exts);
+
+ private:
+  const uint64_t GC_START_LEVEL =
+      20;                      /* Enable GC when < 20% free space available */
+  const uint64_t GC_SLOPE = 3; /* GC agressiveness */
```

Comment:
It would be nice to be able to overwrite the default `GC_SLOPE` and `GC_START_LEVEL` though the ZenFS utility in mkfs.